db/migrate/20250727105116_add_deadline_constraint_to_contests.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddDeadlineConstraintToContests < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_check_constraint :contests,
+                         "deadline IS NOT NULL",
+                         name: "chk_contests_deadline_not_null",
+                         validate: false
```

Comment:
Use `change_column_null`

Reply:
understood, will change this.

Reply:
DONE

Reply:
@salmoneatenbybear  Not an issue since [this](https://github.com/ankane/strong_migrations) is not deployed yet. Refer this on how to skip this error 
```suggestion
    add_check_constraint :contests,
                         "deadline IS NOT NULL",
                         name: "chk_contests_deadline_not_null",
                         validate: false
```

Reply:
Okay I will remake this change.